### PR TITLE
Feat/online stream push

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR does:
1. implement method `Push` for online/mysql
2. add unit test for online/mysql

relate #801

### CODE REVIEW

File: internal/database/online/mysql/store.go [hunk 1]
```diff
@@ -44,8 +46,36 @@ func (db *DB) Purge(ctx context.Context, revisionID int) error {
 	return sqlutil.Purge(ctx, db.DB, revisionID, BackendType)
 }
 
+// TODO: refactor with text/template
 func (db *DB) Push(ctx context.Context, opt online.PushOpt) error {
-	panic("Implement me!")
+	tableName := sqlutil.OnlineStreamTableName(opt.GroupID)
+
+	insertColumns := append([]string{opt.Entity.Name}, opt.FeatureNames...)
+	insertColumnPlaceholers := make([]string, 0, len(insertColumns))
```
Comment (YikSanChan) [inline internal/database/online/mysql/store.go:54]: placeholer -> placeholder

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/online/mysql/store.go (hunk 1 above)
Comment (lianxmfor) [reply internal/database/online/mysql/store.go]: fixed

File: internal/database/online/types.go [hunk 2]
```diff
@@ -27,8 +27,11 @@ type ImportOpt struct {
 }
 
 type PushOpt struct {
-	Entity  *types.Entity
-	Feature types.Feature
+	Entity        *types.Entity
+	EntityValue   string
```
Comment (wfxr) [inline internal/database/online/types.go:31]: EntityValue => EntityKey

Comment (lianxmfor) [reply internal/database/online/types.go]: done

**Pull request merged**